source/mobile/build-custom-mobile-apps.rst

```diff
@@ -0,0 +1,89 @@
+
+This process can be complicated and can greatly increase deployment time, not only initially, but whenever the mobile apps need to be updated. We recommend having your development team `review the Mattermost Mobile Apps developer documentation <https://developers.mattermost.com/contribute/mobile/>`__ to ensure they understand the scale and requirements of taking this path. This documentation provides guidance on building, compiling, signing, and white-labeling Mattermost Mobile apps.
+
+In general, building your own Mobile apps will present some challenges, including:
+
+- Installing the necessary developer tools (such as Nodejs, XCode Developer Tools, Android SDKs, as well as others).
+- Obtaining and providing certificates for your custom Mattermost Mobile apps*.
+- Signing your custom Mattermost Mobile apps*.
+- Distributing your Mobile app to your users.
+
+***** Mattermost Mobile apps are signed, and they have certificates and keys associated with Mattermost and public app stores. This means Mattermost’s Mobile App won’t work if you choose to privately host the Mattermost Push Proxy Service (MPNS). You'll need to build your own custom mobile app.
+
+Custom Mobile App Branding
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ensure you select a unique app name that helps users distinguish your version from others, such as "<Your Company Name> Chat". See our `Brand and Visual Design Guidelines <https://handbook.mattermost.com/operations/operations/company-processes/publishing/publishing-guidelines/brand-and-visual-design-guidelines#name-usage-guidelines.html>`__ in our Company Handbook for details.
+
+Deploying Your Custom App
+-------------------------
+
+You have a number of ways to deploy your custom app for your users to download to their mobile devices.
+
+Using an Enterprise App Store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the most common way for customers to distribute their apps, and it’s our recommended approach. Once your custom app is added to your own Enterprise App Store, your users can download it from the store directly or from an EMM catalog. 
+
+Using an EMM Provider
+~~~~~~~~~~~~~~~~~~~~~
+
+Alternatively, you can use an EMM provider to push the application to the user’s device, then use the AppConfig standard to enforce a selection of app-specific controls. 
+
+For additional details, see the following documentation:
+
+- `Deployment Using an EMM Provider <https://docs.mattermost.com/mobile/deploy-mobile-apps-using-emm-provider.html>`__ to learn more about deploying your custom Mattermost Mobile apps through an EMM provider.
+- `Manage App Configuration Using AppConfig <https://docs.mattermost.com/mobile/mobile-appconfig.html>`__  to learn more about managing your app configuration using App Config.
+- `Mattermost AppConfig Values <https://docs.mattermost.com/mobile/mobile-appconfig.html#mattermost-appconfig-values.html>`__  for information on the configuration options that can be sent from the EMM provider to Mattermost Mobile apps. 
```

Comment:
this links points to the same document as the one before it, maybe group the descriptions ?

Reply:
Great feedback. I'll do exactly that. Thanks!